src/pages/SignUp/SignUp.js

```diff
@@ -1,8 +1,48 @@
 import { Link } from "react-router-dom";
 import { Footer, Navbar } from "../../components";
 import "./SignUp.css";
+import { useReducer } from "react";
+import { signupReducer } from "../../reducers";
+import axios from "axios";
+import { useAuth } from "../../contexts";
```

Comment:
Not mandatory but good to know: You can use path alias to avoid moving back in folders using ../../
refer For Create React App without using Craco section in this https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160

Reply:
Ok I will look into it.